techniques/android/MASTG-TECH-0127.md

```diff
@@ -0,0 +1,20 @@
+--- 
+title: Inspecting an App's Backup Data
+platform: android 
+---
+
+You can inspect an Android app's backup data to verify that sensitive data is not included in the backup. This technique is useful for verifying that the app correctly excludes sensitive data from backups.
+
+## Backup Made via ADB Backup
```

Comment:
This section covers backups made by both ADB Backup and Backup Manager. Should we make this title cover both, or make it more general?

Reply:
You are right, I'll correct that and I need to indicate how to obtain the backup file in the bmgr case